### DISCUSSION TITLE
Woocommerce back order

### DISCUSSION THREAD

**Discussion by QuickBasket** [inspireui/support#5478] [New Feature/Ideas Requests]
Can you please enable backorder feature similar to woocoomerce. I wish to continue to get order for out of stock items as we can procure them at a short notice without the need to stock. Currently I am mangling it through fictitious stock quantity that make my accounting incorrect.

Thanks

**Comment by tryabp** [comment]
is this feature working? As i can test on my woo+fluxstore app, its showing on backorder tag on out of stock products.
